internal/service/apprunner/vpc_connector_test.go

```diff
@@ -167,24 +184,27 @@ resource "aws_vpc" "test" {
   }
 }
 
-resource "aws_security_group" "test" {
-  vpc_id = aws_vpc.test.id
-  name   = %[1]q
+resource "aws_subnet" "test" {
+  cidr_block = "10.1.1.0/24"
+  vpc_id     = aws_vpc.test.id
```

Comment:
Are there specific changes being made here? Or is this just changing the order of the resources in the function?